Content.Shared/Stealth/Components/StealthComponent.cs

```diff
@@ -19,6 +19,12 @@ public sealed partial class StealthComponent : Component
     [DataField("enabled")]
     public bool Enabled = true;
 
+    /// <summary>
+    /// The creature will continue invisible at death.
+    /// </summary>
+    [DataField("enabledOnDeath")]
+    public bool EnabledOnDeath = false;
+
```

Comment:
Is this necessary? Nothing uses it. Just ping me on discord after you see this so we can get this merged.

Reply:
Done. I think